articles/fleet-4.34.0.md

```diff
@@ -0,0 +1,134 @@
+# Fleet 4.34.0 | ChromeOS tables, CIS benchmark load testing.
+
+![Fleet 4.34.0](../website/assets/images/articles/fleet-4.34.0-1600x900@2x.png)
+
+Fleet 4.34.0 is live. Check out the full [changelog](https://github.com/fleetdm/fleet/releases/tag/fleet-v4.33.0) or continue reading to get the highlights.
+For upgrade instructions, see our [upgrade guide](https://fleetdm.com/docs/deploying/upgrading-fleet) in the Fleet docs.
+
+## Highlights
+
+* Fleet adds support for ChromeOS
+* Boosted compliance with 'verified' status
+
+
+### Additional tables for ChromeOS
+
+In line with Fleet's value of 🟢 Results, we work relentlessly to enhance your experience. Our aim is to deliver results, focusing on pragmatic and meaningful improvements. With this in mind, we are delighted to introduce new ChromeOS-specific tables: screenlock, system_state, privacy_preferences, and disk_info. These additions not only represent our commitment to iterative progress but also our dedication to enhancing Fleet's utility for managing and understanding your ChromeOS devices better.
+
+
+### Load testing CIS Benchmarks for macOS
```

Comment:
Is benchmarks capitalized here?

Reply:
'Benchmarks' is capitalized in our [docs](https://fleetdm.com/docs/using-fleet/cis-benchmarks).